apps/meteor/tests/end-to-end/api/01-users.js

```diff
@@ -3732,27 +3125,135 @@ describe('[Users]', function () {
 			agentInfo = await getAgent(agent.user._id);
 			expect(agentInfo).to.have.property('statusLivechat', 'not-available');
 		});
+
+		describe('last owner cases', () => {
+			let room;
+
+			beforeEach(async () => {
```

Comment:
You can probably do both of those updatePermission calls from within a Promise.all

```javascript
beforeEach(() => Promise.all([updatePermission('edit-other-user-active-status', ['admin', 'user']), updatePermission('manage-moderation-actions', ['admin', 'user'])]);
```